crates/doc/src/builder.rs

```diff
@@ -445,7 +445,15 @@ impl DocBuilder {
                 }
             } else {
                 let name = path.iter().last().unwrap().to_string_lossy();
-                let readme_path = Path::new("/").join(&path).display().to_string();
+                let readme_path = Path::new("/")
+                    .join(&path)
+                    .display()
+                    .to_string()
+                    .trim_start_matches('/')
+                    .to_string();
```

Comment:
any chance we can avoid the double allocation here?

Reply:
Change to_string  to to_owned

Reply:
@Evalir Can you check the fix?